lib/libzutil/zutil_import.c

```diff
@@ -52,10 +52,6 @@
 #include <errno.h>
 #include <libintl.h>
 #include <libgen.h>
-#ifdef HAVE_LIBUDEV
```

Comment:
The `include/libzutil.h` header contains some conditional `HAVE_LIBUDEV` blocks which should perhaps be restructured so the prototypes are always declared and function implemented, even if it may always return `ENODATA` or `B_FALSE`.